pkgs/development/compilers/flutter/wrapper.nix

```diff
@@ -4,7 +4,19 @@
 , flutter
 , supportsLinuxDesktop ? stdenv.hostPlatform.isLinux
 , supportsAndroid ? stdenv.hostPlatform.isx86_64
-, includedEngineArtifacts ? null
+, includedEngineArtifacts ? {
```

Comment:
Can we move this out of the ``?``? otherwise changing one value here requires copying all of this.

Reply:
i'm not sure to where
where do you think I should move the default value to?

Reply:
I don't think it's such a big deal. Someone who chooses to configure this will probably not need half of these defaults anyway and simply choose exactly what they need without complex conditionals.